csm/conf/prepare.py

```diff
@@ -118,7 +118,25 @@ def _get_data_nw_info(self):
         except VError as e:
             Log.error(f"Network Validation failed.{e}")
             raise CsmSetupError(f"Network Validation failed.{e}")
-        return data_nw_private_fqdn
+        return [data_nw_private_fqdn]
+    
+    def _get_es_hosts_info(self):
+    	"""
+        Obtains minion data network info
+
+    	:param machine_id: Host Minion id.
```

Comment:
Minion id is not the part params .. remove it from  method doc.. Please add the description of return value in method doc..  Write proper description.

Reply:
Updated required changes in by adding proper description.